src/apps/annotation-image/ImageAnnotationDesktop.tsx

```diff
@@ -252,9 +252,9 @@ export const ImageAnnotationDesktop = (props: ImageAnnotationProps) => {
     setCurrentImage(source);
   }
 
-  const onError = (error) => setToast({
+  const onError = (error: string) => setToast({
```

Comment:
This showed as a type error in my IDE. (The IDE also seemed to be right about it, because the function is used from callback on the sidebar which returns a string rather than an Error object.)